### PR TITLE
Personalization Example with V3 SDK

### LINKED ISSUE #566: Sendgrid v3 with no Client
#### Issue Summary

I have tried to use Sendgrid V3. I do not use Composer. I tried to follow the proposed resolution to issue 412 to no avail. 

I also find the statement that the doc needs to be updated for non client but there is no time, a but lame and discouraging.
 
Do I have to use V3? 
Will V2 be abandoned any time soon?
Can you give  simple step  example of using V3 without a client?

If the answers are Y,Y,N i'm going to have to look for another alternative to Sendgrid.
Rather that having me reproduce the steps that cause the errors, why dont you create and share the steps that dont.

### LINKED ISSUE #567: Sendgrid issue with personalization in V2
#### Issue Summary
I have a simple example of trying to send personalized  emails using substitution in V2.
I created the php program below and get the following output. No emails are sent and the response appears to be empty.

****Output****
Header
{"to":["guy@morazain.com","guy.morazain@gmail.com"],"sub":{"|name|":["Guy","Gaston"],"|password|":["1234","4321"]}}
Content:
Dear |name|,

The information you requested is below
passord: |password|
Params:
Array ( [from] => info@morazain.com [subject] => PHP Test [html] =>
Dear |name|,

The information you requested is below
passord: |password|
[x-smtpapi] => {"to":["guy@morazain.com","guy.morazain@gmail.com"],"sub":{"|name|":["Guy","Gaston"],"|password|":["1234","4321"]}} )
request:https://api.sendgrid.com/api/mail.send.json
print response:

So what am i doing wrong?

****code****
attached
[test2.zip](https://github.com/sendgrid/sendgrid-php/files/1565608/test2.zip)

### PR DESCRIPTION
**Pull request by morazain**
Closes #566
Closes #567

### Checklist
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guide] and my PR follows them.
- [x] I updated my branch with the master branch.
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation about the functionality in the appropriate .md file
- [x] I have added in line documentation to the code I modified

### Short description of what this PR does:
- It provides an working exzample of sending a personalized email to multiple rrecipients
- 

If you have questions, please send an email to [Sendgrid](mailto:dx@sendgrid.com), or file a Github Issue in this repository.

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/sendgrid-php?pullRequest=569) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/sendgrid-php?pullRequest=569) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/sendgrid-php?pullRequest=569) it.</sub>

**Comment by thinkingserious** [comment]
Thank you @morazain! This is on our backlog for review.

File: USE_CASES.md [hunk 1]
```diff
@@ -205,3 +206,106 @@ The file `php.ini` should contain:
...
+# Personalized Emails
+
+```php
+<?php
+/*
+This program demostrates how to send a personalized email
+to multiple recipients using the V3 SDK
+*/
+
+require "../SendGridV3B/sendgrid-php.php";
```
Comment (thinkingserious) [inline USE_CASES.md]: Please make this generic. For example, you can replace SendGridV3B to <Path To>

File: USE_CASES.md [hunk 2]
```diff
@@ -205,3 +206,106 @@ The file `php.ini` should contain:
...
+
+/*
+define  SG_APIKEY and SG_FROM
+or you can create your own instead as follows
+*/
+
+define('SG_APIKEY',"Your api key");
```
Comment (thinkingserious) [inline USE_CASES.md]: Please use this instead:`$apiKey = getenv('SENDGRID_API_KEY');` to promote not including a hard coded API Key in the code. Thanks!

File: USE_CASES.md [hunk 3]
```diff
@@ -205,3 +206,106 @@ The file `php.ini` should contain:
...
+define('SG_FROM',"Your From Email");
```
Comment (thinkingserious) [inline USE_CASES.md]: Your From Email -> from@example.com

**Comment by childish-sambino** [comment]
Closing due to inactivity. Please ping here if work is to continue.

**Pull request closed without merging**